client/src/main/java/io/getstream/chat/android/client/uploader/FileUploader.kt

```diff
@@ -0,0 +1,30 @@
+package io.getstream.chat.android.client.uploader
+
+import io.getstream.chat.android.client.call.Call
+import io.getstream.chat.android.client.utils.ProgressCallback
+import java.io.File
+
+public interface FileUploader {
+
+    public fun sendFile(
+        channelType: String,
+        channelId: String,
+        file: File,
+        callback: ProgressCallback
+    )
+
+    public fun sendFile(channelType: String, channelId: String, file: File): Call<String>
```

Comment:
This interface will be used by third party devs, we shouln't ask them to implement `Call`.
They should return us the final url fo the file uploaded. Internally (Into our `ChatImpl`) we will handle this method and create a `Call` instance